inc/branding/namespace.php

```diff
@@ -363,14 +363,20 @@ function remove_howdy_greeting( WP_Admin_Bar $wp_admin_bar ) {
  * Enqueue branding script for the post previews.
  */
 function enqueue_block_editor_branding_assets() {
+
+	global $pagenow;
+
+	$dependants = [
```

Comment:
It's dependencies rather than dependents, the other way around.

```suggestion
	$dependencies = [
```